instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integration/rum/FragmentTrackingTest.kt

```diff
@@ -101,16 +101,16 @@ internal abstract class FragmentTrackingTest :
         )
 
         // view stopped
-        expectedEvents.add(
-            ExpectedViewEvent(
-                fragmentAViewUrl,
-                3,
-                currentFragmentExtras(activity)
-            )
-        )
+//        expectedEvents.add(
```

Comment:
This expected events is removed. Apparently the JUnit Runner's `callActivityOnStop` doesn't properly forward the fragment's `onStop()` to the fragment lifecycle listeners.